starter-code/src/viking.js

```diff
@@ -1,11 +1,119 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(theDamage) {
+    this.health -= theDamage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super();
```

Comment:
Both here and on the Saxon, if you used the super, you wouldnt need to declare this.health and this.strength ,becasue those properties would come from the parent class.